test/acceptance/multi_tenancy/gql_get_tenant_objects_test.go

```diff
@@ -47,20 +45,23 @@ func TestGQLGetTenantObjects(t *testing.T) {
 			Properties: map[string]interface{}{
 				tenantKey: tenantName,
```

Comment:
hmm, I'm wondering if this should be also removed? bc now the tenantName is stored in Object

Reply:
changed in renaming PR